### PR TITLE
Add roll counter and migrations

### PR DESCRIPTION
**Pull request by kaozaza2**
Add roll counter rateUp for very salty guys like me T-T

### CODE REVIEW

File: 9armbot/player.js [hunk 1]
```diff
@@ -74,6 +74,7 @@ class Player{
             coins: coins,
             status: status,
             exp: 0,
+            rollCounter: 0,
```
Comment (thananon) [inline 9armbot/player.js:77]: How does this work when player's database has already been established? Will this change introduce bugs? How do we migrate from old version of `Player` to here?

**Comment by thananon** [comment]
Good enhancement. Please address my question and let me know if this has been tested or not.

File: 9armbot/player.js (hunk 1 above)
Comment (kaozaza2) [reply 9armbot/player.js:77]: Ok i will add migrate now

**Comment by kaozaza2** [comment]
Not tested now

**Comment by kaozaza2** [comment]
I'll test it later.

**Comment by kaozaza2** [comment]
Tested, work fine.

**Comment by thananon** [comment]
Thank you, please rebase on top of current main.

**Pull request merged**